src/Uno.UI.Wasm/ts/Interop/ManagedObject.ts

```diff
@@ -1,41 +1,18 @@
 ﻿namespace Uno.Foundation.Interop {
 	export class ManagedObject {
 		private static assembly: UI.Interop.IMonoAssemblyHandle;
-		private static dispatchMethod: UI.Interop.IMonoMethodHandle;
+		private static dispatchMethod: any;
```

Comment:
<http://chandermani.com/2017/typescript-anti-pattern-the-any-type/>

Reply:
It seems that exports are not made for delegate-alike, but for full objects. I changed it to be a signature instead.